public/app/ui/components/search/header-search.html

```diff
@@ -1,7 +1,7 @@
 <div class="search header-search">
     <form data-bind="submit: submit">
         <div class="input-group">
-            <input type="text" name="query" placeholder="Search the collections&hellip;" class="form-control" data-bind="textInput: searchText" />
+            <input type="text" name="query" placeholder="Search the collections&hellip;" class="form-control pull-right" data-bind="textInput: searchText" />
```

Comment:
Heh, funny enough I originally wrote "webkit" but then couldn't remember the name of the firefox rendering engine so I changed it to "chrome" since that was the actual browser I'd seen the issue on.